source/features/swap-branches-on-compare.tsx

```diff
@@ -5,7 +5,7 @@ import features from '../libs/features';
 import {getRepoPath, getRepoURL} from '../libs/utils';
 
 function init() {
-	const references = getRepoPath()
+	const references = (getRepoPath() as string)
```

Comment:
It's already typed as `string | undefined`, so would be better to force unwrap it:

```
const references = getRepoPath()!
```